src/Components/blazor/src/Microsoft.AspNetCore.Blazor.Server/Microsoft.AspNetCore.Blazor.Server.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Description>Runtime server features for ASP.NET Core Blazor applications.</Description>
+    <IsProductPackage>true</IsProductPackage>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="$(AspNetCorePackageVersion)" />
+    <PackageReference Include="Microsoft.AspNetCore.WebSockets" Version="$(AspNetCorePackageVersion)" />
+    <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="$(AspNetCorePackageVersion)" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonPackageVersion)" />
+
+    <!-- Used by ws-proxy sources only. Remove this once we're able to consume ws-proxy as a NuGet package. -->
+    <PackageReference Include="Mono.Cecil" Version="0.10.0-beta7" />
```

Comment:
I'm sure this will come up at some point, and it's OK to use an issue to track - but is there a stable version we can use? We don't generally want to depend on prerelease in release, and it looks like that would be easy to happen.

Reply:
Looks like there have been 2 stable releases of Mono.Cecil on nuget since the 0.10.0-beta7 version.  Latest version: [Mono.Cecil 0.10.1](https://www.nuget.org/packages/Mono.Cecil/)